lib/src/sv_sign.c

```diff
@@ -466,6 +466,11 @@ prepare_for_signing(signed_video_t *self)
 SignedVideoReturnCode
 signed_video_add_nalu_for_signing(signed_video_t *self, const uint8_t *bu_data, size_t bu_data_size)
 {
+  if (self->onvif) {
+    int64_t timestamp = 0;  // Use 0 or another valid default timestamp
```

Comment:
No we cannot make up timestamps like this. What would a client say in court if they show a video with a timestamp set to year 1601?
ONVIF requires a valid timestamp. That is the reason why there is only one API. We added timestamps at a later stage. That is why there are two APIs in signed-video-framework.
If no timestamp is set, we should use Axis Signed Video or return an error code to alert the user.